tests/modules/local/filter_assembly.nf.test

```diff
@@ -0,0 +1,33 @@
+nextflow_process {
+
+    name "Test Process FILTER_ASSEMBLY"
+    script "modules/local/filter_assembly.nf"
+    process "FILTER_ASSEMBLY"
+
+    test("Should only filter 2 out of the 3 contigs") {
+
+        when {
+
+            params {
+                min_contig_length = 250
```

Comment:
I thought minimum contig length in the ASA was 500 bases? did we change this?

Reply:
yeah, this is my overriding this to have smaller contigs for the tests